.golangci.yml

```diff
@@ -5,10 +5,27 @@ linters-settings:
     # put imports beginning with prefix after 3rd-party packages;
     # it's a comma-separated list of prefixes
     local-prefixes: github.com/bucketeer-io/bucketeer
+  goheader:
+    # template does not contains the comment indicator '//' or '/*' '*/'
+    template: |-
+      Copyright {{mod-year-range}} The Bucketeer Authors.
```

Comment:
the mod-year-range means a year range that contains the modified date of the file, like 2023-2024 or just 2024.

Reply:
Thank you for your PR!!

We have this script to update the year in the header. We will still need this to update the year, right?
https://github.com/bucketeer-io/bucketeer/blob/main/hack/update-copyright/update-copyright.sh

Reply:
Yes. I missed this script, and I manually updated the files like the ones below.
https://github.com/bucketeer-io/bucketeer/pull/1393/files#diff-a8cf4c0f7d32e71bcb8bb24aa3c2adfa27475fe55b0b0a5831842f95133327feR1

I'll revert such changes and run the script.

Reply:
I fixed these on this commit.
https://github.com/bucketeer-io/bucketeer/pull/1393/commits/8cb823af1de66c8598ac92d137a67fce3ab3942b